tests/run_tests.sh

```diff
@@ -20,28 +20,19 @@ STATUS=0
 
 export HDF5_DISABLE_VERSION_CHECK=1
```

Comment:
So this line doesn't seem necessary. I wonder if it's here from another project...

Reply:
Yeah, this is some straight-up cargo cult stuff from some older version of this script which has been copied and pasted from project to project. Agree with cutting it. (One day this file will be a glorious one liner that just becomes part of the CI/CD directly)